package.json

```diff
@@ -79,6 +79,7 @@
     "@folio/jest-config-stripes": "^2.0.0",
     "@folio/stripes": "^9.0.0",
     "@folio/stripes-cli": "^3.0.0",
+    "@folio/stripes-core": "^10.0.0",
```

Comment:
not clear why you need `@folio/stripes-core` as a dev dependency (even as a just dependency)

Reply:
@NikitaSedyx  I was trying to see if this would fix the github actions error, even though this package should be included in `@folio/stripes`, I will remove it though. The error is below and I don't get any errors when I run the command on local. Do you know why it might be failing?

<img width="968" alt="Screenshot 2024-01-30 at 1 04 11 AM" src="https://github.com/folio-org/ui-plugin-bursar-export/assets/97154067/a1e4e3cd-adae-41d1-807e-3ed0bfe3ec44">